src/LavaNode.cs

```diff
@@ -479,7 +476,7 @@ await _webSocketClient.DisposeAsync()
 
                             await OnTrackStart.Invoke(new TrackStartEventArg {
                                 GuildId = guildId,
-                                EncodedTrack = document.GetProperty("encodedTrack").GetString()
+                                EncodedTrack = document.GetProperty("track").GetProperty("encoded").GetString()
```

Comment:
This should just be a var after event case

Reply:
not following on this one.  what should be a var where?

Reply:
Track.Encoded should be stored in a var similar to guildId

Reply:
is it not? it's being assigned to EncodedTrack

Reply:
nevermind, i think i get it